docs/UserGuide.md

```diff
@@ -338,7 +338,7 @@ Format: `edit -l LESSON_INDEX [n/LESSON_NAME] [c/LESSON_CAPACITY] [p/LESSON_PRIC
 * At least 1 out of 4 optional fields must be provided.
 
 Examples:
-* `edit -l c/20 p/80`
+* `edit -l 1 c/20 p/80` edits the 1st lesson's capacity and price to 20 and 80 respectively.
```

Comment:
It may be more complete to phrase as such: `updates the 1st lesson's capacity to 20 students and price to $80`